scripts/generate_forger_keys.sh

```diff
@@ -25,31 +25,38 @@ done
 
 # Checking if .env file exist and sourcing
 env_file_exist "${ROOT_DIR}/${ENV_FILE}"
-source "${ROOT_DIR}/${ENV_FILE}" || { echo "Error: could not source ${ROOT_DIR}/${ENV_FILE} file. Fix it before proceeding any further.  Exiting..."; exit 1; }
+SCNODE_ROLE="$(grep 'SCNODE_ROLE=' "${ROOT_DIR}/${ENV_FILE}" | cut -d '=' -f2)" || { echo "SCNODE_ROLE value is wrong. Check ${ROOT_DIR}/${ENV_FILE} file"; exit 1; }
 SCNODE_REST_PORT="$(grep 'SCNODE_REST_PORT=' "${ROOT_DIR}/${ENV_FILE}" | cut -d '=' -f2)" || { echo "SCNODE_REST_PORT value is wrong. Check ${ROOT_DIR}/${ENV_FILE} file"; exit 1; }
 export SCNODE_REST_PORT
 select_compose_file
 
 # Checking if init.sh script was executed or not
 scnode_wallet_seed="$(grep 'SCNODE_WALLET_SEED=' "${ROOT_DIR}/${ENV_FILE}" | cut -d '=' -f2)" || { echo "SCNODE_WALLET_SEED value is wrong. Check ${ROOT_DIR}/${ENV_FILE} file"; exit 1; }
 if [ -z "${scnode_wallet_seed}" ]; then
-  fn_die "The stack was not yet initialized. Run init.sh script first. There is nothing to start.  Exiting ..."
+  fn_die "The stack was not yet initialized. The wallet seed has not been populated correctly. Please run init.sh script or check your  "${ROOT_DIR}/${ENV_FILE}" file. Exiting ..."
 fi
 
-# Generate Vrf Key (forger keys)
-VRF_KEY="$($COMPOSE_CMD -f ${compose_file} exec "${CONTAINER_NAME}" gosu user curl -s -X POST "http://127.0.0.1:${SCNODE_REST_PORT}/wallet/createVrfSecret" -H "accept: application/json" -H 'Content-Type: application/json' | jq .result[].publicKey)" 
+# Checking if the evm node is running 
+if [ -n "$(docker ps -q -f status=running -f name="${CONTAINER_NAME}")" ]; then
+  echo "Checking if the node is reachable..."
```

Comment:
I think you can get rid of this message since the function right below has its own ECHO messages

Reply:
Kinda ; if the node is not reachable, you will just see black screen and no response for a long time (long enough to think something is wrong), that's why I added it.